.mocharc.json

```diff
@@ -1,10 +1,10 @@
 {
   "diff": true,
-  "extension": "spec.ts",
+  "extension": ".ts",
```

Comment:
Why was `spec` removed?

Reply:
because of unit test file name ends with `.test.ts`, the file name is `utility.test.js`. Anyway, as we decided to remove validation and test. I'll set it back to `spec.ts`. Thank you!